ios/RNLinksdk.m

```diff
@@ -772,6 +772,28 @@ + (NSString *)stringForEventName:(PLKEventName *)eventName {
             return @"FAIL_OAUTH";
         case PLKEventNameValueHandoff:
             return @"HANDOFF";
+        case PLKEventNameValueIdentityVerificationStartStep:
```

Comment:
Quick question: it looks like you checked for any missing events (not just IDV ones), is that correct?

Reply:
Yes. I added all of the missing cases for events & views.